.github/workflows/npm-publish-cli.yml

```diff
@@ -21,12 +21,6 @@ jobs:
       - name: Build the CLI
         run: yarn workspace @storyblok/field-plugin-cli build
       - name: Publish the package to NPM
-        run: cd packages/cli && npm publish --access public --tag beta
-        env:
-          NODE_AUTH_TOKEN: ${{secrets.NPM_PUBLISH}}
```

Comment:
I'm wondering if this token would be still necessary, @eunjae-lee. 🤔

It was kept in the library workflow.

Reply:
I'm not sure if I understand your question correctly, but this token is still used within publish.ts to publish the library to NPM.